app/src/main/res/drawable/mentor_spinner_border.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    android:shape="rectangle">
+    <corners android:radius="6dip" />
+    <stroke
+        android:color="#E35058"
+        android:width="@android:dimen/dialog_min_width_minor" />
```

Comment:
Changing this to a number should fix the error when inflating the MentorFragment.